storefront/app/controllers/workarea/storefront/categories_controller.rb

```diff
@@ -7,5 +7,11 @@ def show
       raise InvalidDisplay unless model.active? || current_user.try(:admin?)
       @category = Storefront::CategoryViewModel.new(model, view_model_options)
     end
+
+    private
+
+    def page_specific_content_policy
```

Comment:
This would be nicer as a delegation, something like `delegate :content_security_policy_config, to: :@category`

Reply:
I guess? Not 💯 on what you mean.

Reply:
We can rename this method (and the one on the model) to `content_security_policy_config` or something similar and use the above delegation rather than define the "extra" method here.